src/resssources/HeaderParagraph.rs

```diff
@@ -0,0 +1,10 @@
+pub struct HeaderParagraph {
+    format: &str,
+    upsteam_name: Option<&str>,
+    upstream_contact: Option<&str>,
+    source: Option<&str>,
+    disclaimer: Option<&str>,
+    comment: Option<&str>,
+    license: Option<Licence>,
+    copyright: Option<&str>,
```

Comment:
could be a struct Copyright {name, year: ({u8, u8}, u8), email }

Reply:
Yeah but most of them will be optional as the DEP5 doesn't enforce a format for the copyright field